server/tsconfig.dev.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "module": "commonjs",
```

Comment:
Do we still need this?

Reply:
This is the only tiny amount of non-es6 config only used for dev purposes.  Can't be removed yet.  You can see it referenced in package.json in mocha/dev run-scripts.

Reply:
It's actually a little ironic .. we moved to es6 modules when we did in part so that we could have a non-rebuild dev environment for the browser. Now we want a non-rebuild dev environment for node .. we need cjs modules.

Dammit js, get your act together.